### DISCUSSION TITLE
How to use svglite instead svg for R graphics output

### DISCUSSION THREAD

**Discussion by mavam** [quarto-dev/quarto-cli#2293] [Q&A] [answered]
For several output formats, the option `fig-format` controls the output type of the graphics. I am coming from the R and currently use `svg` but would like to use a different package to render the SVGs: [svglite](https://svglite.r-lib.org/):

> Compared to the built-in [`svg()`](https://rdrr.io/r/grDevices/cairo.html), svglite produces smaller files, and leaves text as is, making it easier to edit the result after creation. It also support multiple nice features such as embedding of web fonts.

It seems there's no direct way to control this from the outside. How would I go after this? Is a customer filter a path to a solution? Or would I have to hack deeper in the code?

**Comment by mcanouil** [comment, marked as answer]
The following in your header should do it:

```yaml
fig-format: svg
knitr:
  opts_chunk: 
    dev: "svglite"
```

**Reply by mavam** [reply]
Works like a charm!
